tools/gfastats/gfastats.xml

```diff
@@ -188,6 +194,22 @@
                             label="Report actual sequence" help="It reports also the actual sequence"/>
                     </when>
                 </conditional>
+                <conditional name="locale_condition">
+                    <param name="locale_selector" type="select" label="Locale">
```

Comment:
wow, isn't that too detailed? Should we just pick utf-8 as default and foce users to use it?

Reply:
hmm I think it would still be useful to have the non-utf8, because that output would be machine-readable without having to take the commas out... maybe the one with commas should be default then, since it's more human-readable, and then have an option for the comma-less output?

Reply:
latest commit is trying it this way, with utf-8 as default and a toggle to turn it off 👍🏼 it does look more straightforward!